processor/blockprocessor/initialize.go

```diff
@@ -105,18 +117,20 @@ func fullNodeCatchup(logger *log.Logger, round basics.Round, catchpoint, dataDir
 	for status.LastRound < round {
 		time.Sleep(2 * time.Second)
```

Comment:
Generally, it lets you do something like this to terminate early:
```suggestion
		select {
			case <- ctx.Done():
				return
			case <- time.After(2 * time.Second):
		}
```

Reply:
We would also want to use `defer node.Stop()` in case we exit early.